Kwenta/kwenta
# Add a new price feed for atomic swap

## Description
The atomic swap and the fee reclamation may use different exchange rates. We want to use the different price feed for the different swap types.

## Related issue
#1051

## Motivation and Context
Show the more accurate amount that the user could receive when the atomic swap could be triggered.

## How Has This Been Tested?
1. Open the market page and switch to L1
2. Choose two different currencies except for sLink
3. See the amount that the user will receive on UI and compare it with https://dev.kwenta.io/exchange

## Screenshots (if appropriate):
